compose.yml

```diff
@@ -0,0 +1,29 @@
+services:
+  frontend:
+    container_name: frontend
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+    environment:
+      - GOVUK_APP_DOMAIN="localhost"
+      - GOVUK_WEBSITE_ROOT=http://localhost/
+      - RAILS_ENV=production
+      - PREVIEW_MODE=true # Set to true to enable preview mode and prevent redirecting to the https://www.gov.uk/trade-tariff
```

Comment:
this comment out of context doesn't make too much sense. I knew you mean you're preventing a redirect we expect in prod env